hieradata_aws/common.yaml

```diff
@@ -64,6 +64,9 @@ node_class: &node_class
   ckan:
     apps:
       - ckan
+  ckan29:
```

Comment:
I don't get this, this Hiera change is instructing that there will be a new class of machine `ckan29` and install `ckan`.

Reply:
Haven't read through the code, I don't think **any** change to **this** file is needed.

Reply:
There is some logic in the `ckan` app which deploys files according to the environment so no need to create a `ckan29` app. I did try this route initially but hit a number of issues.

Reply:
i don't see the need for this code. can you point to the logic/code which uses this hiera?